assets/css/_variables.scss

```diff
@@ -10,34 +10,22 @@ $global-font-weight: 400;
 $global-line-height: 1.5rem;
 
 // Color of the background
-$global-background-color: #fff !default;
-$global-background-color-dark: #292a2d !default;
-$global-background-color-black: #000000 !default;
+$global-background-color: var(--global-background-color) !default;
 
 // Color of the text
-$global-font-color: #161209 !default;
-$global-font-color-dark: #a9a9b3 !default;
-$global-font-color-black: #d9d9d9 !default;
+$global-font-color: var(--global-font-color) !default;
 
 // Color of the secondary text
-$global-font-secondary-color: #4d4d4d !default;
-$global-font-secondary-color-dark: #b5b5b5 !default;
-$global-font-secondary-color-black: #a9a9b3 !default;
+$global-font-secondary-color: var(--global-font-secondary-color) !default;
 
 // Color of the link
-$global-link-color: #161209 !default;
-$global-link-color-dark: #a9a9b3 !default;
-$global-link-color-black: #d9d9d9 !default;
+$global-link-color: var(--global-link-color) !default;
 
```

Comment:
I have been thinking about this, at this point do we really a mapping from CSS variables to SCSS variables? Is there anything stopping us from just writing

```css
.style {
  color: var(--color);
}
```

Reply:
I did this at first, but then I kept a mapping. Manage all colors in one SCSS file. I think it's easy to advance and retreat.
This would be useful for refactoring to reduce the same color variables, and to modify them using a format similar to "-- primary".